src/main/java/examples/Examples.java

```diff
@@ -410,4 +410,24 @@ public void example12(MongoService mongoService) {
 
   }
 
+  public void example13_0(MongoService mongoService) {
+
+    JsonObject document = new JsonObject().put("title", "The Hobbit")
+        .put("publicationDate", new JsonObject().put("$date", -1018659600000L /* 21/9/1937 */));
```

Comment:
Maybe I am misreading them or missing something but according to the Mongo docs, the date should be represented as:
{ "$date" : { "$numberLong" : "<dateAsMilliseconds>" } }
But in the above code it seems to be represented as:
{ "$date" : "<dateAsMilliseconds>"  }
Could you clarify?